### PR TITLE
Add RMBG notebook

### PR DESCRIPTION
**Pull request by eaidova**
Ticket: CVS-136657

### CODE REVIEW

File: notebooks/rmbg-background-removal/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Background removal with RMBG v1.4 and OpenVINO
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/eaidova/openvino_notebooks_binder.git/main?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fopenvinotoolkit%252Fopenvino_notebooks%26urlpath%3Dtree%252Fopenvino_notebooks%252Fnotebooks%2F291-rmbg-background-removal%2F291-rmbg-background-removal.ipynb)
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/291-rmbg-background-removal/291-rmbg-background-removal.ipynb)
```
Comment (eaidova) [inline notebooks/rmbg-background-removal/README.md:3]:
```suggestion
[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/eaidova/openvino_notebooks_binder.git/main?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fopenvinotoolkit%252Fopenvino_notebooks%26urlpath%3Dtree%252Fopenvino_notebooks%252Fnotebooks%2Frmbg-background-removal%2Frmbg-background-removal.ipynb)
[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/master/notebooks/rmbg-background-removal/rmbg-background-removal.ipynb)
```

**Comment by eaidova** [comment]
<p>added</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1839/discussion/#comment-2020985820'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by eaidova** [comment]
> View / edit / reply to [this conversation](https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1839/discussion/#comment-2020985850) on ReviewNB
> 
> _aleksandr-mokrov commented on 2024-03-26T16:57:32Z_ ----------------------------------------------------------------
> 
> Line #1.    from briarmbg import BriaRMBG
> 
> It requires more files
> 
> ----> 5 from .MyConfig import RMBGConfig
> 
> ImportError: attempted relative import with no known parent package

model code was updated ~8 hours ago, before that it worked, I updated notebook, @aleksandr-mokrov  please take a look one more time

**Review by itrushkin — APPROVED** [review]
LGTM

**Pull request merged**